### ISSUE TITLE
godoc?

### ISSUE THREAD

**Issue by dhiemaz** [kitabisa/perkakas#28]
Is there any Godoc (golang documentation) for this repo ?

**Comment by fairyhunter13** [comment]
@dhiemaz Yeah, I think we need Godoc too for this repository. How about contribute to this repository?
